container/images/base/fetch_repo_and_collect_coverage.sh

```diff
@@ -49,6 +49,13 @@ else
 fi
 # Output: Repo available at ${LOCAL_REPO_DESTINATION}
 
+# Convert from Dos to Unix - in case format is in non-unix format
+echo "Convert from Dos to Unix - in case format is in non-unix format"
+find ${LOCAL_REPO_DESTINATION} -type f           \
```

Comment:
We need to create a failing test. The input that we have it is too big or a small focused test.
How about we craft a failing test by adding another commit (+tag) to the Hmmm repo that will cause the compute coverage test to fail. The commit should add a bash script with CRLF that will fail unless processed.
You can get creative: https://github.com/julianghionoiu/tdl-runner-hmmm